src/components/Menu/menuRules.ts

```diff
@@ -6,12 +6,12 @@ const solidBorder = (color: string) => ({
 
 export default {
   root: ({ props, variables }) => {
-    const { type, shape, vertical } = props
+    const { fluid, type, shape, vertical } = props
     return {
       display: 'flex',
       ...(vertical && {
         flexDirection: 'column',
-        width: pxToRem(200),
+        ...(!fluid && { width: pxToRem(200) }),
```

Comment:
just thinking aloud - shouldn't the Menu take all the available width of the parent container by default? This seems to be a reasonable thing given that otherwise there are no ways to customize `Menu`'s width - according to this code line.

Reply:
Not sure about that, this is 1:1 from SIUR.

Reply:
yes, but if put SUI aside - doesn't it seem to be a problem? FYI @levithomason